utils/node_local_test.py

```diff
@@ -0,0 +1,798 @@
+#!/usr/bin/python3
+
+"""Test code for dfuse"""
+
+import os
+import sys
+import time
+import uuid
+import json
+import signal
+import subprocess
+import tempfile
+import pickle
+
+from collections import OrderedDict
+
+class DFTestFail(Exception):
+    """Used to indicate test failure"""
+    pass
+
+def umount(path):
+    """Umount dfuse from a given path"""
+    cmd = ['fusermount', '-u', path]
```

Comment:
shouldn't this be "fusermount3" as the packages installed are "fuse3" in JenkinsFile?

Reply:
Good catch.  It seems the fuse3 package brings in fuse so fusermount will be present but I'll change this.